src/apriltag/apriltag/apriltag_node.py

```diff
@@ -41,9 +42,12 @@ def reset_callback(self, request, response):
     """Publishes the tag if it exists"""
 
     def postTransform(self, tag):
-        if tag and (self.get_clock().now().to_msg().sec == self.averagedTag.header.stamp.sec):
+        if tag and (self.get_clock().now().to_msg().sec == tag.header.stamp.sec):
             self.get_logger().info(str("Resetting the odom"))
-            self.tf_broadcaster.sendTransform(tag)
+            req = SetPose.Request()
+            req.pos = [tag.transform.translation.x, tag.transform.translation.y, tag.transform.translation.z]
+            req.orient = [tag.transform.rotation.x, tag.transform.rotation.y, tag.transform.rotation.z]
+            self.cli_set_zed_pose.call_async(req)
```

Comment:
I don't actually know if this is the "right" way to do what we want or if broadcasting a TF is better, testing is definitely needed for this.